cmd/src/server.rs

```diff
@@ -861,6 +901,24 @@ fn check_system_config(config: &TiKvConfig) {
     }
 }
 
+fn try_lock_conflict_addr<P: AsRef<Path>>(existed_path: P) -> File {
+    let f = File::create(existed_path.as_ref()).unwrap_or_else(|e| {
+        fatal!(
+            "failed to create lock at {}: {}",
+            existed_path.as_ref().display(),
+            e
+        )
+    });
+
+    if f.try_lock_exclusive().is_err() {
+        fatal!(
+            "lock {} failed, maybe another instance is binding with this address.",
```

Comment:
The error message may be confusing.

Reply:
I don't know where it is confusing, change to `binding with same port` ?

Reply:
Fixed, and I will add `merged` tag.